apps/judicial-system/web/src/components/Cards/CasesCard/index.tsx

```diff
@@ -1,7 +1,7 @@
 import { FC } from 'react'
 
 import { Box, LinkV2, LoadingDots, Text } from '@island.is/island-ui/core'
-import { CaseTableType } from '@island.is/judicial-system/types'
+import { CaseTableType } from '@island.is/judicial-system-web/src/graphql/schema'
```

Comment:
Generally, we want to import generated GraphQL if they exist. I have made a number of similar changes elsewhere.